src/pages/account/security/sudo.tsx

```diff
@@ -47,63 +56,63 @@ const getServerSideProps: GetServerSideProps = async (context) => {
         connectUnreachable: ERRORS_DATA.CONNECT_UNREACHABLE,
         notFound: ERRORS_DATA.NOT_FOUND,
       };
 
-      throw webErrorFactory(webErrors.notFound);
+      // throw webErrorFactory(webErrors.notFound);
 
-      // const userCookie = await getServerSideCookies<UserCookie>(request, {
-      //   cookieName: "user-cookie",
-      //   isCookieSealed: true,
-      //   cookieSalt: config.cookieSalt,
-      // });
+      const userCookie = await getServerSideCookies<UserCookie>(request, {
+        cookieName: "user-cookie",
+        isCookieSealed: true,
+        cookieSalt: config.cookieSalt,
+      });
 
-      // if (userCookie) {
-      //   const primaryIdentities = await getIdentities(
-      //     config.managementCredentials,
-      //     userCookie.sub,
-      //   )
-      //     .then((identities) => {
-      //       return identities.filter((identity) => {
-      //         return (
-      //           identity.primary &&
-      //           (identity.type == IdentityTypes.EMAIL.toLowerCase() ||
-      //             identity.type == IdentityTypes.PHONE.toLowerCase())
-      //         );
-      //       });
-      //     })
-      //     .catch((error) => {
-      //       if (error instanceof GraphqlErrors) {
-      //         throw webErrorFactory({
-      //           ...webErrors.identityNotFound,
-      //           parentError: error,
-      //         });
-      //       }
+      if (userCookie) {
+        const primaryIdentities = await getIdentities(
+          config.managementCredentials,
+          userCookie.sub,
+        )
+          .then((identities) => {
+            return identities.filter((identity) => {
+              return (
+                identity.primary &&
+                (identity.type == IdentityTypes.EMAIL.toLowerCase() ||
+                  identity.type == IdentityTypes.PHONE.toLowerCase())
+              );
+            });
+          })
+          .catch((error) => {
+            if (error instanceof GraphqlErrors) {
+              throw webErrorFactory({
+                ...webErrors.identityNotFound,
+                parentError: error,
+              });
+            }
 
-      //       if (error instanceof ConnectUnreachableError) {
-      //         throw webErrorFactory({
-      //           ...webErrors.connectUnreachable,
-      //           parentError: error,
-      //         });
-      //       }
+            if (error instanceof ConnectUnreachableError) {
+              throw webErrorFactory({
+                ...webErrors.connectUnreachable,
+                parentError: error,
+              });
+            }
 
-      //       throw error;
-      //     });
+            throw error;
+          });
 
-      //   if (primaryIdentities.length > 1) {
-      //     return {
-      //       props: {
-      //         primaryIdentities,
-      //       },
-      //     };
-      //   }
-      // }
+        if (primaryIdentities.length > 1) {
+          return {
```

Comment:
Shouldn't this be `> 0`? We can have a case where the user only have 1 primary identity.